codespell_lib/data/dictionary.txt

```diff
@@ -9318,6 +9358,7 @@ istead->instead
 istener->listeners
 ists->its, lists,
 isue->issue
+isused->is used
```

Comment:
I think `issued` should be the first result here.

Reply:
Right. Fixed.